README.md

```diff
@@ -29,4 +29,9 @@ function mystery(n) {
 
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
-might help with the notation for mathematical expressions.
+might help with the notation for mathematical expressions. 
+
+The solution is 3t(n/3)+O(n^5), I solved this by first looking at the recurive calls that are made which is three. Then looking at the nested loop
+it is n^2*n*n^2 which equals n^5.
```

Comment:
Your recurrence relation is correct. But please prove this by using backward substitution method.

Reply:
I thought It said would not need a formal proof so I will add that.